scripts/ubuntu-bartender/ubuntu-bartender

```diff
@@ -509,7 +535,24 @@ build-provider-create $bartender_name
   then
     # copy the release notes tools to all the hooks directories
     find livecd-rootfs/live-build/ -type d -name '*hooks*' |
-      xargs -I {} sh -c 'mkdir --parents {}/extra/release-notes-tools && cp --archive --force release-notes-tools/* {}/extra/release-notes-tools/'
+      xargs -I {} sh -c "mkdir --parents {}/extra/release-notes-tools && cp --archive --force $temp_dir/release-notes-tools/* {}/extra/release-notes-tools/"
```

Comment:
is it sbom tool or release notes tool? those are 2 different things. but approving anyway...

Reply:
Apologies. I snook in a fix for adding `$temp_dir`